src/logic.js

```diff
@@ -0,0 +1,56 @@
+import objects from './objects';
+import dom from './domStuff';
+
+const projectExists = (title) => {
+  const data = localStorage;
+  let exists = true;
+  if (!data[title]) {
+    exists = false;
+  }
+  return exists;
```

Comment:
[OPTIONAL]
Code be shortened to `return !data[title] ? false : true;`

Reply:
Try and see other instances where code like this code be shortened

Reply:
Thank you for the advice! 👏